tests/console/aaa_base.pm

```diff
@@ -53,4 +53,8 @@ sub run {
     assert_script_run "cd ; rm -rf /tmp/aa_base_test";
 }
 
+sub test_flags {
+    return {no_rollback => 1} if get_var('PUBLIC_CLOUD');
```

Comment:
I have the feeling this is dangerous. As in none PUBLIC_CLOUD case you would return `undef` which isn't what other code expect see https://github.com/os-autoinst/os-autoinst/blob/master/basetest.pm#L118

I didn't checked all of your `no_rollback` statements. But I have the feeling that we could do it in `lib/consoletest.pm`, and change with all the `tests/console/*` modules, right?

Something like this:
```
sub tests_flags {
     return get_var('PUBLIC_CLOUD') ? { no_rollback => 1 } : {};
}
```

Reply:
That it good idea, I will try that tonight.

Reply:
Well, the `tests_flags` is overwritten most of the time.

Reply:
Well @cfconrad there was `tests_flags` instead of `test_flags` so let's watch [it](http://pdostal-server.suse.cz/tests/5187) now